src/main/pages/che-7/administration-guide/proc_collecting-che-metrics-with-prometheus.adoc

```diff
@@ -5,15 +5,13 @@ This section describes how to use the Prometheus monitoring system to collect, s
 
 .Prerequisites
 
-* {prod-short} is exposing metrics on port `8087`. See xref:enabling-and-exposing-{prod-id-short}-metrics_{context}[].
+* {prod-short} is exposing metrics on port `8087`. See xref:enabling-and-exposing-{prod-id-short}-metrics_{context}[Enabling and exposing {prod-id-short} metrics].
```

Comment:
This shouldn't be required -- as long as an `xref` works, it supplies the link title automatically.

Reply:
Thanks for the feedback. I am looking into it.